doc/classes/Basis.xml

```diff
@@ -37,14 +37,15 @@
 			<param index="0" name="axis" type="Vector3" />
 			<param index="1" name="angle" type="float" />
 			<description>
-				Constructs a pure rotation basis matrix, rotated around the given [param axis] by [param angle] (in radians). The axis must be a normalized vector.
+				Constructs a [Basis] that only represents rotation, rotated around the [param axis] by the given [param angle], in radians. The axis must be a normalized vector.
+				[b]Note:[/b] This is the same as using [method rotated] on the [constant IDENTITY] basis. With more than one angle consider using [method from_euler], instead.
 			</description>
 		</constructor>
 		<constructor name="Basis">
 			<return type="Basis" />
 			<param index="0" name="from" type="Quaternion" />
 			<description>
-				Constructs a pure rotation basis matrix from the given quaternion.
+				Constructs a [Basis] that only represents rotation from the given [Quaternion].
```

Comment:
It might be worth mentioning that the conversion from Quaternion->Basis->Quaternion is reversible, but the conversion from Basis->Quaternion->Basis is irreversible because Quaternion does not properly account for scale.

Currently Godot sometimes causes unexpected transforming when converting from Basis to Quaternion with negative scales, but I haven't investigated if there is a way to handle this well.

Reply:
The "Basis to Quaternion" quirk should be written in Quartenion's constructor. I 'll keep that in mind.